loginsightwebhookdemo/vrealizeorchestrator.py

```diff
@@ -7,17 +7,19 @@
 
 __author__ = "John Dias"
 __license__ = "Apache v2"
-__version__ = "1.1"
+__version__ = "1.2"
 
 
-# vRealize Orchestrator server workflow hostname (or hostname:port)
+# vRealize Orchestrator server workflow hostname:port (default port is 8281)
 VROHOSTNAME = ''
-
+# For some labs, using self-signed will result in error during request due to cert check
+# flip this flag to False to bypass certificate checking in those cases
+VERIFY = True
 
 
 @app.route("/endpoint/vro/<WORKFLOWID>", methods=['POST'])
-@app.route("/endpoint/vro/<WORKFLOWID>/<ALERTID>", methods=['PUT'])
```

Comment:
vRO requires a body in the workflow request that includes inputs.  Even if there are no inputs, the payload is required.  So, a shim is needed between vR Ops and vRO.  What I did in this update was refactor to have the LI -> vRO alerts handled as they were before the vR Ops capability was added.  The vR Ops alert messages are different, so the parsed JSON doesn't match up to the LI function.

Does that help?